added the release note for @import supports(…) (#27796)
mdn/content: files/en-us/mozilla/firefox/releases/115/index.md

### CSS

- The CSS {{cssxref("animation-composition")}} property is now supported by default. You can use this property to specify the composite operation to use when multiple animations affect the same property simultaneously. ([Firefox bug 1823862](https://bugzil.la/1823862)).

> - The CSS {{cssxref("@import")}} [at-rule](/en-US/docs/Web/CSS/At-rule) `supports(…)` conditions are now supported in (CSS) import rules (`@import supports(...)`). Which allows stylesheets to be imported if the included feature is supported in the users browser. ([Firefox bug 1830779](https://bugzil.la/1830779))

Reviewer: In the bug, they say "Turn on supports conditions in CSS imports (@import supports(...)) by default.".

I assume they're referring to the `supports-condition` parameter in the `supports()` function ([formal syntax]((https://developer.mozilla.org/en-US/docs/Web/CSS/@import#formal_syntax))) Or is this referring to the `supports()` function itself? Please change the suggested text as appropriate.

```suggestion
- The `supports-conditions` in the CSS {{cssxref("@import")}} [at-rule](/en-US/docs/Web/CSS/At-rule) `supports()` function is now supported by default. This feature allows stylesheets to be imported only if the specified feature is supported in the user's browser. ([Firefox bug 1830779](https://bugzil.la/1830779)).
```
